app/view/templates/homeopt.php

```diff
@@ -22,12 +22,10 @@
                     <legend>Sort</legend>
                     <div>
                         <select name="sortby" id="sortby">
-                            <?php
-                            foreach (Wcms\Opt::SORTBYLIST as $col) {
-                                $name = Wcms\Model::METADATAS_NAMES[$col];
-                                echo '<option value="' . $col . '" ' . ($opt->sortby() == $col ? "selected" : "") . '>' . $name . '</option>';
-                            }
-                            ?>
+                            <?php foreach (Wcms\Opt::SORTBYLIST as $col) :
+                                $name = Wcms\Model::METADATAS_NAMES[$col]; ?>
```

Comment:
```suggestion
                            <?php foreach (Wcms\Opt::SORTBYLIST as $col) : ?>
                                <?php $name = Wcms\Model::METADATAS_NAMES[$col]; ?>
```
or maybe this

Reply:
whatever, here I should have send better vars from the controller. like a good old assoc array ready to be used.

see #459

Reply:
Meanwhile, this last one, more verbose but properly indented, seems to be better.